sunpy/net/hek/hek.py

```diff
@@ -18,11 +23,15 @@
 from sunpy.time import parse_time
 from sunpy.util import dict_keys_same, unique
 from sunpy.util.xml import xml_to_dict
+from sunpy import __file__
 
 __all__ = ['HEKClient', 'HEKTable', 'HEKRow']
 
 DEFAULT_URL = 'https://www.lmsal.com/hek/her?'
+UNIT_FILE_PATH = Path(os.path.dirname(__file__)) / "net" / "hek"/ "unit_properties.json"
+COORD_FILE_PATH = Path(os.path.dirname(__file__)) / "net" / "hek"/ "coord_properties.json"
 
+u.add_enabled_aliases({"steradian": u.sr, "arcseconds": u.arcsec, "degrees": u.deg, "sec": u.s})
```

Comment:
I think this will need to be added to the function that converts the units and then have it remove the aliases. We don't want these to be permanent.